src/modal/modal.js

```diff
@@ -454,6 +456,7 @@ angular.module('ui.bootstrap.modal', [])
                 backdropClass: modalOptions.backdropClass,
                 windowClass: modalOptions.windowClass,
                 windowTemplateUrl: modalOptions.windowTemplateUrl,
+                container: $modalProvider.defaultContainer,
```

Comment:
`container` is just coming off of my use of `defaultContainer`. Would you want `appendTo` and `defaultContainer`? Or change both to `appendTo` and `defaultAppendTo`?

Or perhaps something else entirely?